cl/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "courtlistener-webpack-loader",
+  "version": "1.0.0",
+  "description": "use modern js frameworks with courtlistener",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "tests"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "scripts": {
+    "build": "NODE_ENV=production ./node_modules/.bin/webpack --config webpack.config.js",
+    "dev": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --hot",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
Can you just skim this list for anything that you don't recognize and purge it? I know I installed a bunch of things along the way that I thought might be useful, but I don't know if I was good about actually removing them again.